README.md

```diff
@@ -87,6 +87,8 @@ A curated list of awesome resources, learning materials, tools, frameworks, plat
   * [Advanced Topics in Types and Programming Languages, B. Pierce](https://www.amazon.com/dp/0262162288).
   * [Building an Optimizing Compiler, B. Morgan](https://www.amazon.com/dp/155558179X).
   * [Compiling with Continuations, A. Appel](https://www.amazon.com/dp/052103311X).
+  * [Design Concepts in Programming Languages, F. Turbak, D. Gifford, M. Sheldon](https://www.amazon.com/dp/0262201755) - also contains some introductory level material.
+  * [The Garbage Collection Handbook: The Art of Automatic Memory Management, R. Jones, A. Hosking, E. Moss](https://www.amazon.com/dp/1420082795).
```

Comment:
Can you move this book further down in the list right before `The SSA Book` resource to maintain the alphabetical order?